tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Can we move it to `package.json`?

Reply:
Unfortunately, no, Typescript doesn't allow it. And this file is required for typecheck: it points out the files Typescript should work with.

Reply:
Can we move it to `test/`?

Reply:
It affects all `d.ts` files, so we should leave it in the root.